packages/integration/src/lib/search/search-results-tool/search-results-tool.component.ts

```diff
@@ -525,11 +525,15 @@ export class SearchResultsToolComponent implements OnInit, OnDestroy {
     }
   }
 
-  toggleTopPanel() {
-    if (this.topPanelState === 'expanded') {
-      this.topPanelState = 'collapsed';
+  toggleTopPanel(event?: MouseEvent) {
```

Comment:
You can uncommit this changed file (it's already in another PR)

Reply:
ok